src/Jackett/JackettModule.cs

```diff
@@ -59,6 +59,10 @@ protected override void Load(ContainerBuilder builder)
             Mapper.CreateMap<WebClientByteResult, WebClientStringResult>().ForMember(x => x.Content, opt => opt.Ignore()).AfterMap((be, str) =>
             {
                 str.Content = Encoding.UTF8.GetString(be.Content);
+                if(str.Content.Contains("NorBits"))

```

Comment:
From my experience most trackers are not UTF8. I had similar problems with German trackers.
Initial I intended to look at the headers but I soon came across trackers sending the wrong header or no header at all. I ended up doing the decoding inside the indexer code, see my post below for an example.